docs/imviz/plugins.rst

```diff
@@ -223,6 +223,10 @@ catalog dropdown menu.
     This plugin is still under active development. As a result, the search only uses the SDSS DR17 catalog
     and works best when you only have a single image loaded in a viewer.
 
+To load a catalog from an ecsv file, choose "From File..." and choose a valid file.  The file must be able
```

Comment:
The way you wrote it, it can load anything can `astropy.table.QTable` can parse, which is more than ECSV.
```suggestion
To load a catalog from a file, choose "From File..." and choose a valid file.  The file must be able
```

Reply:
I'll wait on this for now until we converge on how general the parser should be for now (if we keep it specific for now, maybe this should change to "a JWST ecsv catalog file" instead).

Reply:
(updated change log to be specific to JWST for now... we'll defer generalizing for a follow-up effort, if necessary)